File: src/MovieRental/Rental.cs
namespace MovieRental;

/// <summary>
/// Represents a customer renting a movie.
/// </summary>
public class Rental
{
    private Movie _movie;
    private int _daysRented;

    public Rental(Movie movie, int daysRented)
    {
        _movie = movie;
        _daysRented = daysRented;
    }

    public Movie Movie => _movie;

    public int DaysRented => _daysRented;
}

File: src/Rental.js
/**
 * The rental class represents a customer renting a movie.
 */
class Rental {

    constructor(movie, daysRented) {
        this._movie = movie;
        this._daysRented = daysRented;
    }

    getDaysRented() {
        return this._daysRented;
    }

    getMovie() {
        return this._movie;
    }
}

export default Rental;
